src/patternfly/components/Tabs/__tabs-list.hbs

```diff
@@ -29,13 +32,18 @@ __tabs-list--IsCloseDisabled: bool
     __tabs-item--aria-label="Users"
     __tabs-item--text="Users"
     }}
-  {{> __tabs-item
-    __tabs-item--current="true"
+  {{#> wrapper
     __tabs-item--id="containers"
     __tabs-item--icon-name="fas fa-box"
     __tabs-item--aria-label="Containers"
     __tabs-item--text="Containers"
     }}
+    {{#if __tabs-list--IsOverflowSelected}}
+      {{> __tabs-item }}
+    {{else}}
+      {{> __tabs-item __tabs-item--current="true"}}
+    {{/if}}
```

Comment:
Yep, you can set it to something that doesn't exist - typically we use `reset`, so `__tabs-list--IsOverflowSelected=reset`. https://github.com/patternfly/patternfly/blame/main/src/patternfly/components/Form/examples/Form.md#L346

You can also declare `newcontext` on a partial and it will unset any inherited props. https://github.com/patternfly/patternfly/blame/main/src/patternfly/components/Menu/examples/Menu.md#L172

Reply:
I think we might have had a bit of miscommunication about the reason I went with the implementation I had here, or there was some connection that I just wasn't making with your suggestions.

I did rewrite this in a better way in [44b78c5](https://github.com/patternfly/patternfly/pull/4876/commits/44b78c530e9952ec375ca568a9f99ef20441f4a0) with the aid of a simple helper I wrote to inverse a boolean though. Let me know what you think!